ledger/api-server-damlonx/reference/src/main/scala/com/daml/ledger/api/server/damlonx/reference/ReferenceServer.scala

```diff
@@ -21,6 +21,29 @@ import scala.util.Try
 object ReferenceServer extends App {
   val logger = LoggerFactory.getLogger(this.getClass)
 
+  final case class Config(
+      port: Int,
+      portFile: Option[File],
+      archiveFiles: List[File],
+  )
+
+  val argParser = new scopt.OptionParser[Config]("reference-server") {
+    head("DamlOnX Reference Server")
```

Comment:
Add a sentence describing what the reference server does and how it starts by default. Along the lines of:

```
A fully compliant DAML Ledger API server backed by an in-memory store. Due to its lack of persistence it is not meant for production, but to serve as a blueprint for other DAML Ledger API server implementations.
```